src/components/component-view/component-view.js

```diff
@@ -8,28 +8,36 @@ import Icon from "../icon/icon"
 import loadable from "@loadable/component"
 
 import "./component-view.scss"
+import viewerData from "../../data/component-viewer.yaml"
 
 import { Notification } from 'bootstrap-italia'
 
+
 const slugify = require('slugify')
 
 const SyntaxHighlighter = loadable(() => import('./syntax-highlighter'))
 
 const ComponentView = ({
-  name,
+  variantName,
   source,
   content,
-  viewer,
   idPrefix,
   viewerHeight,
   accordionOpen,
   accordionShow,
-  accordionLabel,
   accordionUrl,
-  accordionSrLabel,
   accordionSrCopyLabel,
+  componentViewerData
 }) => {
 
+  if (componentViewerData?.variants) { // it is not a component viewer... 
+    content = componentViewerData.variants.filter(item => item.name === variantName)[0].content;
```

Comment:
This is a runtime error if the source YAML doesn't contain `variants` (or matching variants), which we don't get until it's too late, in production.

Not too sure about the context, but maybe we can use something along the lines of

```js
content = componentViewerData.variants.filter(item => item.name === variantName)[0]?.content;
```

provided that having an `undefined` content works as expected later on in the code. Otherwise we need to handle it someway.